### ISSUE TITLE
Icon highlighted

### ISSUE THREAD

**Issue by BitFros7y** [Francesco149/weebp#5]
Whenever i click at empty space on "desktop" (wallpaper/video) previous icon is highlighted (outlined with border).

------------------
System Information
------------------
Time of this report: 12/12/2018, 19:10:54
       Machine name: ICE-PC
   Operating System: Windows 7 Ultimate 64-bit (6.1, Build 7601) Service Pack 1 (7601.win7sp1_ldr_escrow.180422-1430)
           Language: English (Regional Setting: English)
System Manufacturer: Gigabyte Technology Co., Ltd.
       System Model: P31-ES3G
               BIOS: Award Modular BIOS v6.00PG
          Processor: Pentium(R) Dual-Core  CPU      E5200  @ 2.50GHz (2 CPUs), ~2.5GHz
             Memory: 2048MB RAM
Available OS Memory: 2046MB RAM
          Page File: 2284MB used, 1808MB available
        Windows Dir: C:\Windows
    DirectX Version: DirectX 11
DX Setup Parameters: Not found
   User DPI Setting: Using System DPI
 System DPI Setting: 96 DPI (100 percent)
    DWM DPI Scaling: Disabled
     DxDiag Version: 6.01.7601.17514 32bit Unicode

![untitled](https://user-images.githubusercontent.com/2947012/49889221-3d535180-fe41-11e8-8ea6-1937a7365507.png)

**Comment by kmgb** [comment]
This can be closed as this is standard behaviour of Windows and is unrelated to weebp.

**Comment by BitFros7y** [comment]
No, standard windows behaviour is to deselect everything, and no borders/outline/shadow is shown on any icon. So basically if i click anywhere on empty desktop space there should not be any borders or anything selected.

**Comment by kmgb** [comment]
This outline occurs when you select an icon on the desktop and use the arrow keys to move the selection, then click anywhere else. You can use Refresh on the desktop and it will disappear temporarily, but it will reappear next time you focus the desktop.

Are you able to reproduce it without using arrow keys?

**Comment by BitFros7y** [comment]
Yes, thats the problem, even if you dont use arrows at all, that outline is shown, all you need to do is click anywhere on desktop that has video as background. Im not on the same computer anymore, but i will try to download latest weebp and reproduce issue on my laptop.
